$OpenBSD: patch-src_PovConfig_h,v 1.1 2019/03/02 20:05:29 cwen Exp $
ports-gcc fix for
src/PovConfig.cpp:678:37: error: 'system' was not declared in this scope
Index: src/PovConfig.h
--- src/PovConfig.h.orig
+++ src/PovConfig.h
@@ -36,6 +36,7 @@
 #include <qheader.h>
 #include <qtooltip.h>
 #include <qwhatsthis.h>
+#include <stdlib.h>
 #include "k3dsurf.h"
 
 class PovConfig: public QMainWindow
